internal/olm/operator/registry/operator_installer_test.go

```diff
@@ -16,22 +16,233 @@ package registry
 
 import (
 	"context"
+	"fmt"
+	// TODO(asmacdo) why doesnt vimgo do this for me
+	"os"
 
+	// "github.com/fatih/color"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	v1 "github.com/operator-framework/api/pkg/operators/v1"
 	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	// apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	// "k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/sets"
 	crclient "sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 
 	"github.com/operator-framework/operator-sdk/internal/olm/operator"
+	"github.com/operator-framework/operator-sdk/internal/util/k8sutil"
 )
 
+type FakeCatalogCreator struct {
+	client crclient.Client
+}
+
+func (f FakeCatalogCreator) CreateCatalog(ctx context.Context, name string) (*v1alpha1.CatalogSource, error) {
```

Comment:
missing comment because it is an exported func. Also, why it needs to be exported?

Reply:
It doesnt. nice catch